src/main/java/eu/europeana/metis/sandbox/common/amqp/RecordMessageConverter.java

```diff
@@ -23,17 +24,19 @@
   private static final String COUNTRY = "country";
   private static final String STATUS = "status";
   private static final String STEP = "step";
+  private static final String EXCEPTION = "exception";
 
   private static final Charset DEFAULT_CHARSET = StandardCharsets.UTF_8;
 
   @Override
   public Message toMessage(Object object, MessageProperties messageProperties)
       throws MessageConversionException {
```

Comment:
This exception is a RuntimeException and does not need to be explicitly mentioned here?

Reply:
True, it was included because the interface has it and the IDE inherited when doing the override, I will remove it